### ISSUE TITLE
tsUnit.ts external compiling error

### ISSUE THREAD

**Issue by haskellcamargo** [Steve-Fenton/tsUnit#5]
The external `tsUnit.ts` has a type mismatch error on line 260, that is raised  on compile and has the following message:

```
tsUnit.ts(260,37): error TS2345: Argument of type 'Event' is not assignable to parameter of type 'HashChangeEvent'.
```

I believe this is an internal error of the library caused by a bad type signature.

**Comment by Steve-Fenton** [comment]
Hi haskellcamargo - thank you for raising this. It looks like a breaking change in the tsd file. I will fix this shortly. Thanks.

**Comment by Steve-Fenton** [comment]
Fixed in v1.8.0. Available now on NuGet.

https://www.nuget.org/packages?q=tsUnit

**Comment by RossBille** [comment]
How about the npm version?

**Comment by Steve-Fenton** [comment]
That has been published too - v1.8.0.
